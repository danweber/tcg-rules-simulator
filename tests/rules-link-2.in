VERSION:2

# link rules check

FIELD1:Multi,CS3-06,PLUG,CS3-04,CS3-02,CS3-04,CS3-02
HAND1:CS3-11 CS3-04 CS3-02
GAME:P1 T3 MAIN 5
#GET-X-PLAYS
DUMP:DP,KEYWORDS,HAND,TRASH

ECHO: -- evo into thing without link +3
ECHO: -- we should get to select the things we trash
ECHO: -- also the order things go into trash doesn't make sense

CMD:EVO CS3-11 Multi
DUMP:DP,KEYWORDS,HAND,TRASH

EXIT
