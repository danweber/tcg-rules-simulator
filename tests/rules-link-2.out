FIELD1:Multi,CS3-Shieldwing,PLUG,CS3-Arcanetweet,CS3-Fairyfeatherlink,CS3-Arcanetweet,CS3-Fairyfeatherlink,9500DP,DODGE,LINK [-+]\D
FIELD2:
TRASH1:
TRASH2:
HAND1:CS3-Illumifalcon CS3-Arcanetweet CS3-Fairyfeatherlink
HAND2:
GAME:P1 T3 MAIN 5
ECHO: -- evo into thing without link +3
ECHO: -- we should get to select the things we trash
ECHO: -- also the order things go into trash doesn't make sense
FIELD1:Multi,CS3-Illumifalcon,CS3-Shieldwing,PLUG,CS3-Fairyfeatherlink,10000DP,DODGE
FIELD2:
TRASH1:CS3-Arcanetweet CS3-Arcanetweet CS3-Fairyfeatherlink
TRASH2:
HAND1:CS3-Arcanetweet CS3-Fairyfeatherlink
HAND2:
GAME:P1 T3 MAIN 1
